### ISSUE TITLE
Sorting labels

### ISSUE THREAD

**Issue by Akotkar23** [ultralytics/yolov5#7744]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Can we sort individual ID (1,2,3,4) as labels 
Sharing the code below: 
import torch
import cv2
import numpy as np
from PIL import Image
model = torch.hub.load('D:/yolov5-master', 'custom', path='C:/Users/admin/Downloads/SSLabels/Graphs/MDCount.pt', source='local')
img = 'C:/Users/admin/BagDetection/SS Images/2.jpg'
results = model(img)
results.render()
results.show()

Cannot do the changes in detect.py as the changes doesnot reflect while using hub.load. Does anyone have an idea?
@glenn-jocher is it possible to do the changes in the above code? Thanks.
![sort](https://user-images.githubusercontent.com/68338164/167540028-74a71142-bf6c-49b7-ae39-c8ca284d2954.png)

**Comment by glenn-jocher** [comment]
@Akotkar23 hi buddy, I'm not sure exactly what you're asking here. The results are in a torch tensor, i.e. `results.xywh` that you can sort any way you'd like.

**Comment by Akotkar23** [comment]
> @Akotkar23 hi buddy, I'm not sure exactly what you're asking here. The results are in a torch tensor, i.e. `results.xywh` that you can sort any way you'd like.

Thanks, I'll try using 'results.xywh' and Want to know that can I achieve this using the code I've mentioned in the question for sorting.
